rpc_jobs/rpc_octavia.yml

```diff
@@ -43,7 +78,7 @@
     # currently we only do master and pike. Once RPC-O switches
     # to Queens we will retire rpc-octavia and use the
     # upstream installer
-    branches:
+    branch:
```

Comment:
This needs to be a single value and there needs to be another PM job for the other branch. 'branch' is not a list, but a string.

Reply:
Never mind. I'm confused. Branch is an axis.